index.html

```diff
@@ -1292,10 +1292,11 @@
 
 ### Ancillary uses
 
-[=Sites=] sometimes use data in ways that aren't needed for the user's immediate
+[=Sites=] sometimes use data in ways that aren't needed for the user's primary
 goals. For example, they might bill advertisers, measure site performance, or
-tell developers about bugs. These uses are known as <dfn data-lt="ancillary
-use">ancillary uses</dfn>.
+tell developers about bugs. These uses, which provide indirect support for,
```

Comment:
I'd prefer

```
These uses might provide indirect support for the user's primary goals, and are known as
<dfn data-lt="ancillary use">ancillary uses</dfn>.
```

I appreciate this might seem a very minor change, but one major concern around "ancillary data" is that the user agent is sharing a bunch of information about the user and user's behavior with site, without anyway of knowing whether its in the user's interest.

Reply:
We could add modifiers like can/might/may/often to "provide" if the intention is that "ancillary" should cover uses that don't support the user's goals at all, even indirectly; but I think the current sentence structure does a better job of actually clarifying what we mean by "ancillary" for two reasons: it connects up the term more directly with the defining phrase, and it contrasts the ancillary uses with essential uses to clarify the relationship between the two categories.